tests/app/test_base.py

```diff
@@ -1,262 +1,164 @@
-from unittest.mock import patch, call
-
 import pytest
+from pytest_mock import MockerFixture
 
 from corva.app.base import BaseApp
-from corva.app.base import ProcessResult
-from corva.event.base import BaseEvent
 from corva.event.data.base import BaseEventData
-from corva.utils import get_state_key
+from corva.event.event import Event
+from tests.conftest import ComparableException, APP_KEY, CACHE_URL
+
+
+@pytest.fixture(scope='function')
+def base_app(mocker: MockerFixture):
+    # patch __abstractmethods__, so we can initialize BaseApp
+    mocker.patch.object(BaseApp, '__abstractmethods__', set())
```

Comment:
worth a better descriptionof why do we patch abstractmethods?

Reply:
Expanded the comment